addons/mail/static/src/components/chatter_container/chatter_container.js

```diff
@@ -56,15 +60,14 @@ export class ChatterContainer extends Component {
      * @private
      */
     async _insertFromProps(props) {
-        const messaging = await this.env.services.messaging.get();
```

Comment:
I'm not comfortable with this remove, if init messaging is slower than form open I think this can lead to crash without the await